cmd/restic/cmd_check.go

```diff
@@ -479,3 +500,11 @@ func selectRandomPacksByFileSize(allPacks map[restic.ID]int64, subsetSize int64,
 	packs := selectRandomPacksByPercentage(allPacks, subsetPercentage)
 	return packs
 }
+
+type checkResult struct {
+	MessageType     string   `json:"message_type"`
```

Comment:
Add a comment that the expected message type is `summary`.

Reply:
Done